docs/guides/file-uploads.md

```diff
@@ -0,0 +1,125 @@
+---
+id: file-uploads
+title: File Uploads
+sidebar_label: File Uploads
+---
+
+You can transfer files between client and server using GraphQL. It is really simple as in REST. If you're using `express-graphql` as your GraphQL Server, you should install **[graphql-upload](https://github.com/jaydenseric/graphql-upload)** package. It has two things we need to add to our GraphQL Application.
+
+## Adding Upload Feature to our GraphQL Server
+
+You should extend your existing server in order to make it able to handle file uploads. Check the following recipes for GraphQL Express and Apollo Server.
+
+### Adding GraphQL Upload Middleware to GraphQL Express
+
+Let's assume we have something like the following at the beginning, and as you can see we add `graphqlUploadExpress` middleware into `/graphql` route to make it handle file uploads and pass it through the resolvers.
+
+```ts
+import { GraphQLModule } from '@graphql-modules/core';
+import * as express from 'express';
+import * as graphqlHTTP from 'express-graphql';
+
+const { schema } = new GraphQLModule({
+    /*...*/
+});
+
+const app = express();
+
+app.use(
+    '/graphql',
+    graphqlUploadExpress({ maxFileSize: 10000000, maxFiles: 10 }),
+    graphqlHTTP({ schema, graphiql: true })
+  );
+
+app.listen(4000);
+```
+
+### Enabling uploads in Apollo-Server
+
+The only thing you need to have for Apollo Server is to add `uploads` field to your server configuration.
```

Comment:
Add something like `if you are using apollo-server, you don't have to add the express middleware, you just need to add this to your configuration:`